client/default.conf

```diff
@@ -35,18 +34,31 @@ server {
         proxy_pass http://localhost:3000/api;
     }
 
+    location /env {
+        if ( $request_method !~ ^(GET)$ ) {
+           return 405;
+        }       
+        proxy_set_header        Host $host;
+        proxy_set_header        X-Real-IP $remote_addr;
+        proxy_set_header        X-Forwarded-Host $host;
+        proxy_pass http://localhost:3000/env;
+    }
+
     location /share {
         proxy_set_header        Host $host;
         proxy_set_header        X-Real-IP $remote_addr;
         proxy_set_header        X-Forwarded-Host $host;
         proxy_pass http://localhost:3000/share;
-    }
+    }   
```

Comment:
trailing ws. There are other places in this file also I believe that doesn't show up in this diff

tip: https://github.com/jonase/.emacs.d/blob/master/init.el#L67 ;)